src/ORM/DataList.php

```diff
@@ -2061,4 +2061,14 @@ public function chunkedFetch(int $chunkSize = 1000): iterable
             $currentChunk++;
         }
     }
+
+    /**
+     * Prepopulate any extension caches with the current dataclass and IDs of records
+     */
+    public function prepopulateCaches(): void
+    {
+        $dataClass = $this->dataClass;
+        $ids = $this->column('ID');
+        $this->extend('updatePrepopulateCaches', $dataClass, $ids);
```

Comment:
`update` usually means "please make changes to at least one of the arguments I'm passing in" - but in this case that's not what is intended. It should probably be `onPrepopulateCaches` instead i.e. "We're prepopulating caches now, please perform any cache prepopulation you need to do"
```suggestion
        $this->extend('onPrepopulateCaches', $dataClass, $ids);
```

For precedent see e.g. `onInit` in various controller `init()` methods

Reply:
Updated